src/main/java/com/google/sps/servlets/ListOrganizationsServlet.java

```diff
@@ -72,25 +74,47 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public Query getQueryFromParams(HttpServletRequest request, GivrUser currentUser) {
     Query query = new Query("Distributor").addSort("creationTimeStampMillis", SortDirection.DESCENDING);
 
+    ArrayList<Filter> filterCollection = new ArrayList<Filter>();
+
+    String zipcode  = "";
+    boolean queryForZipcode = false;
+    if (request.getParameter("zipcode") != null) {
+      zipcode = request.getParameter("zipcode");
+      queryForZipcode = true;
```

Comment:
Do you want to verify this zipcode? (It's OK to say no, as the search will fail, but consider what will help the user understand and fix their mistake)

Reply:
I was thinking we could display "No Results Found" if nothing comes back for the query, which would apply if they enter a bad zip code. I added a point to the filtering frontend issue / todo's list to address this